docs/Update wallet section (#1252)
filecoin-project/filecoin-docs: content/en/about-filecoin/managing-assets/index.md

---
title: "Managing assets"

> description: "Many interactions with the Filecoin network require FIL to process. This page explains the different denominations of FIL, how to store it, and how to aquire it."

Reviewer: acquire

> description: "Many interactions with the Filecoin network require FIL to process. Additionally, participants in hackathons may receive FIL as payment for bounties and prizes. This page explains the different denominations of FIL, how to store it, and how to acquire it."

Reviewer 2: ```suggestion
description: "FIL is the native token of the Filecoin network. This page explains how to acquire, store, and use FIL tokens."
```
```suggestion
description: "Many interactions with the Filecoin network require FIL to process. Additionally, participants in hackathons may receive FIL as payment for bounties and prizes. This page explains the different denominations of FIL, how to store it, and how to acquire it."
```

Simplify a lot. Also, while hackathon payments is one use-case that's not well-served by the current docs, it should not be called out at the network documentation level.

> description: "Many interactions with the Filecoin network require FIL to process. Additionally, participants in hackathons may receive FIL as payment for bounties and prizes. This page explains the different denominations of FIL, how to store it, and how to acquire it."

Reviewer 2: ```suggestion
description: "FIL is the native token of the Filecoin network. This page explains how to acquire, store, and use FIL tokens."
```
```suggestion
description: "Many interactions with the Filecoin network require FIL to process. Additionally, participants in hackathons may receive FIL as payment for bounties and prizes. This page explains the different denominations of FIL, how to store it, and how to acquire it."
```

Simplify a lot. Also, while hackathon payments is one use-case that's not well-served by the current docs, it should not be called out at the network documentation level.

## Wallets

<!--TODO: request out to Johnny to verify what goes in relref for the link in this section-->

> Wallets provide a way to securely store digital assets. `FIL` is a native token, meaning it runs on its own blockchain and is not compatible with all wallets out of the box. [Wrapped FIL]{{< relref "wrapped-filecoin" >}} allows the token to interact with other blockchains if desired.

Reviewer 2: ```suggestion
`FIL` is the native token of the Filecoin network. FIL tokens are used as an economic incentive to ensure data is stored reliably over time.

Wallets provide a way to securely store digital assets. Here are some wallets with FIL support that you might consider.

Note: If you are already running your own lotus node, you can also [manage FIL wallets from the command line](https://lotus.filecoin.io/lotus/manage/manage-fil/).

```
```suggestion
Wallets provide a way to securely store digital assets. `FIL` is a native token, meaning it runs on its own blockchain and is not compatible with all wallets out of the box. [Wrapped FIL]{{< relref "wrapped-filecoin" >}} allows the token to interact with other blockchains if desired.
```
Let's leave the wrapped FIL reference out, people can find it in the sidebar if desired.

> Wallets provide a way to securely store digital assets. `FIL` is a native token, meaning it runs on its own blockchain and is not compatible with all wallets out of the box. [Wrapped FIL]{{< relref "wrapped-filecoin" >}} allows the token to interact with other blockchains if desired.

Reviewer 2: ```suggestion
`FIL` is the native token of the Filecoin network. FIL tokens are used as an economic incentive to ensure data is stored reliably over time.

```
```suggestion
Wallets provide a way to securely store digital assets. `FIL` is a native token, meaning it runs on its own blockchain and is not compatible with all wallets out of the box. [Wrapped FIL]{{< relref "wrapped-filecoin" >}} allows the token to interact with other blockchains if desired.
```
Let's leave the wrapped FIL reference out, people can find it in the sidebar if desired.

> ### Custodial vs non-custodial

Reviewer 2: ```suggestion
### Custodial vs. non-custodial
```
```suggestion
### Custodial vs non-custodial
```

> ### Custodial vs non-custodial

Reviewer 2: ```suggestion
### Custodial vs. non-custodial
```
```suggestion
### Custodial vs non-custodial
```

> ### Custodial vs non-custodial

Reviewer 2: Now that I see the whole article, I think we can remove this section on custodial vs non-custodial. It's extremely well written! But less immediately relevant for novice users, and more expert users can find this information elsewhere.

> ### Custodial vs non-custodial

Reviewer 2: Now that I see the whole article, I think we can remove this section on custodial vs non-custodial. It's extremely well written! But less immediately relevant for novice users, and more expert users can find this information elsewhere.

Author: Section removed.

A custodial wallet is one which is hosted by an exchange. These wallets can come with a greater sense of security as the exchange holds the private keys. Many will also provide insurance to help recover funds lost to fraud, similar to centralized stock or bank accounts. The tradeoff is assets in custodial wallets can be frozen or seized by the associated exchange. Most custodial wallets also have KYC (Know Your Customer) requirements which require users to verify their identity in order to use the service.

With non-custodial wallets, the user holds their own private key and has total control over the movement of assets. Non-custodial wallet users are also solely responsible for securing their assets. Protecting private keys and seed phrases is extremely important to prevent fraudulent access to assets. Most non-custodial wallets do not have KYC requirements and allow users to maintain anonymity.

> ### Hot vs cold

Reviewer 2: ```suggestion
### Hot vs. cold
```
```suggestion
### Hot vs cold
```

> ### Hot vs cold

Reviewer 2: ```suggestion
### Hot vs. cold
```
```suggestion
### Hot vs cold
```

> Hot wallet refers to any wallet which is connected to a network. They can be mobile, desktop, or browser based. Connecting to a network comes with its own security concerns, therefore it is recommended to keep large balances in cold wallets and use hot wallets to hold funds you need to access frequently.

Reviewer 2: ```suggestion
Hot wallet refers to any wallet which is permanently connected to the internet. They can be mobile, desktop, or browser based. Hot wallets make it faster and easier to access digital access, but being could be vulnerable to online attacks. Therefore, it is recommended to keep large balances in cold wallets and only use hot wallets to hold funds you need to access frequently.
```
```suggestion
Hot wallet refers to any wallet which is connected to a network. They can be mobile, desktop, or browser based. Connecting to a network comes with its own security concerns, therefore it is recommended to keep large balances in cold wallets and use hot wallets to hold funds you need to access frequently.
```

> Cold wallet most commonly refers to hardware wallet devices. Accessing a cold wallet typically requires physical possession of the device plus knowledge of the private key which makes them more resistant to loss via theft. Transaction signing happens within the device itself and private keys are never exposed to the network. Cold wallets can take longer to access and are most useful for storing larger balances securely. 

Reviewer 2: ```suggestion
Cold wallets most commonly refer to hardware wallet devices shaped like a USB stick. They are typically offline, and only connected to the internet for transactions. Accessing a cold wallet typically requires physical possession of the device plus knowledge of the private key, which makes them more resistant to loss via theft. Cold wallets can be less convenient, and are most useful for storing larger balances securely. 
```
```suggestion
Cold wallet most commonly refers to hardware wallet devices. Accessing a cold wallet typically requires physical possession of the device plus knowledge of the private key which makes them more resistant to loss via theft. Transaction signing happens within the device itself and private keys are never exposed to the network. Cold wallets can take longer to access and are most useful for storing larger balances securely. 
```

> Cold wallet most commonly refers to hardware wallet devices. Accessing a cold wallet typically requires physical possession of the device plus knowledge of the private key which makes them more resistant to loss via theft. Transaction signing happens within the device itself and private keys are never exposed to the network. Cold wallets can take longer to access and are most useful for storing larger balances securely. 

Reviewer 2: ```suggestion
Cold wallets most commonly refer to hardware wallet devices shaped like a USB stick. They are typically offline, and only connected to the internet for transactions. Accessing a cold wallet typically requires physical possession of the device plus knowledge of the private key, which makes them more resistant to loss via theft. Cold wallets can be less convenient, and are most useful for storing larger balances securely. 
```
```suggestion
Cold wallet most commonly refers to hardware wallet devices. Accessing a cold wallet typically requires physical possession of the device plus knowledge of the private key which makes them more resistant to loss via theft. Transaction signing happens within the device itself and private keys are never exposed to the network. Cold wallets can take longer to access and are most useful for storing larger balances securely. 
```

### Security

Wallets that have gone through an audit have had their codebase checked by a recognized security firm for security vulnerabilities and potential leaks. However, just because a wallet has had an audit does not mean that it’s 100% bug-proof. Be incredibly cautious when using unaudited wallets.

Never share your seed phrase, password, or private keys. Bad actors will often use social engineering tactics such as phishing emails or posing as customer service or tech support to lure users into handing over their private key or seed phrase.

> Wallets allow you to manage your assets without having to meet the [relatively high specifications required to run a Lotus node]({{< relref "set-up" >}}) for Filecoin. Here are some wallets that you might consider.

Reviewer 2: ```suggestion
```
```suggestion
Wallets allow you to manage your assets without having to meet the [relatively high specifications required to run a Lotus node]({{< relref "set-up" >}}) for Filecoin. Here are some wallets that you might consider.
```

> Wallets allow you to manage your assets without having to meet the [relatively high specifications required to run a Lotus node]({{< relref "set-up" >}}) for Filecoin. Here are some wallets that you might consider.

Reviewer 2: ```suggestion
```
```suggestion
Wallets allow you to manage your assets without having to meet the [relatively high specifications required to run a Lotus node]({{< relref "set-up" >}}) for Filecoin. Here are some wallets that you might consider.
```

> We do not provide support for any of these wallets. We recommend that you use an audited wallet that comes with hardware-wallet integration, such as Glif web wallet.

Reviewer 2: ```suggestion
We do not provide support for any of these wallets.
```
```suggestion
We do not provide support for any of these wallets. We recommend that you use an audited wallet that comes with hardware-wallet integration, such as Glif web wallet.
```

This isn't a great recommendation for first-time token users. Let's remove any recommendation at all, and make sure the list & rest of article are helpful.

> We do not provide support for any of these wallets. We recommend that you use an audited wallet that comes with hardware-wallet integration, such as Glif web wallet.

Reviewer 2: ```suggestion
We do not provide support for any of these wallets.
```
```suggestion
We do not provide support for any of these wallets. We recommend that you use an audited wallet that comes with hardware-wallet integration, such as Glif web wallet.
```

This isn't a great recommendation for first-time token users. Let's remove any recommendation at all, and make sure the list & rest of article are helpful.

>  Name                                           | Description                                                                                                                          | Audited |

Reviewer 2: Let's move the wallets list above the tradeoffs (hot vs cold, security, etc.)

Reviewer 2: Also, let's make the text in this list more user-friendly. Can wait til 2nd round of revisions.

Reviewer 2: Also, let's make the text in this list more user-friendly. Can wait til 2nd round of revisions.

Author: Moved list up in doc

>  Name                                           | Description                                                                                                                          | Audited |

Reviewer 2: Let's move the wallets list above the tradeoffs (hot vs cold, security, etc.)

<!--TODO: request out to Johnny to verify what goes in relref for the link in this section-->
Wallets provide a way to securely store digital assets. `FIL` is a native token, meaning it runs on its own blockchain and is not compatible with all wallets out of the box. [Wrapped FIL]{{< relref "wrapped-filecoin" >}} allows the token to interact with other blockchains if desired.

> ### Custodial vs. non-custodial

Reviewer 3: ```suggestion
### Custodial versus non-custodial
```

| [MathWallet](https://mathwallet.org/en-us/)    | A multi-currency wallet.                                                                                                             | Unknown |
| [Trust wallet](https://trustwallet.com/)                                   | A multi-currency wallet. This is the official wallet of Binance.                                                                     | Unknown |

> ### Hot vs. cold

Reviewer 3: ```suggestion
### Hot versus cold
```
